waspc/packages/deploy/src/providers/fly/setup/setup.ts

```diff
@@ -109,6 +110,24 @@ Press any key to continue or Ctrl+C to cancel.`);
 		replaceLineInLocalToml(minMachinesOptionRegex, 'min_machines_running = 1');
 	}
 
+	if (!doesLocalTomlContainLine(internalPortOptionRegex)) {
+		await question(`\n⚠️  There was an issue setting up your server app.
+We tried modifying your server fly.toml to set ${boldText(
+		'internal_port = 8080',
+	)}, but couldn't find the option ${boldText(
+	'internal_port',
+)} in the fly.toml.
+
+This means your server app might not be accessible.
```

Comment:
I got confused by this part, what are we trying to say here: if the `internal_port` option is not present in the Fly TOML file the server app might not work correctly because Fly won't know how to reach it?